.github/workflows/oppiabot.yml

```diff
@@ -1,22 +0,0 @@
-# Contains jobs corresponding to Oppiabot.
```

Comment:
Ditto for this & the other files in .github/--these aren't relevant to your PR and may have been merged incorrectly. Another thought: is your fork up-to-date with the latest develop?